src/PerformanceTests/Common/Scenarios/SendLocalOneOnOne/SendLocalOneOnOneRunner.cs

```diff
@@ -16,9 +16,7 @@ partial class SendLocalOneOnOneRunner : BaseRunner
 
     protected override void Start()
```

Comment:
Let's go all out, as @danielmarbach says. Worst case scenario is that we might learn something ;-)